lib/birthday_reader.rb

```diff
@@ -1,12 +1,11 @@
+require 'yaml'
+
 class BirthdayReader
   def self.get_birthdays(file_path)
     birthdays = []
     if File.exist?(file_path) 
-      file = File.new(file_path, 'r')
-      file.each_line do |line|
-        birthdays << line.chomp.split
-      end
-      file.close
+	bday_list = YAML.load_file(file_path)
```

Comment:
The primary reason I switched to a structured format for the database was to drill down to the exact date to fetch all the birthdays on the day. What would be the use case to have return all the birthdays together? It would just use more memory (for no particular gain, whatsoever) IMHO.